TextileIPFSNativeModule.js

```diff
@@ -60,7 +60,7 @@ export default {
   },
 
   getPhotos: async function (offset: ?string, limit: number, thread: string): string {
-    const result = await TextileIPFS.getPhotos(offset, limit, thread)
+    const result = await TextileIPFS.getPhotos(offset || "", limit, thread)
```

Comment:
saw a couple error related to receiving `null`s in go layer here, but maybe misleading - anyway, the empty string is expected in the go lib.